src/Altinn.Notifications.Persistence/Migration/v0.09/02-setup-functions.sql

```diff
@@ -0,0 +1,46 @@
+DROP FUNCTION IF EXISTS notifications.getorder_includestatus(_alternateid UUID, _creatorname TEXT);
+CREATE OR REPLACE FUNCTION notifications.getorder_includestatus(_alternateid UUID, _creatorname TEXT)
+RETURNS TABLE (
+    alternateid UUID,
+    creatorname TEXT,
+    sendersreference TEXT,
+    created TIMESTAMPTZ,
+    requestedsendtime TIMESTAMPTZ,
+    processed TIMESTAMPTZ,
+    processedstatus orderprocessingstate,
```

Comment:
Doesn't changing the type require changes in code?

Reply:
We should have integration tests that run this procedure and verify the response content. It that's green change shouldn't be breaking and require alterations in the code.